testing/test_unittest.py

```diff
@@ -828,3 +828,64 @@ def test_should_not_run(self):
     """)
     reprec = testdir.inline_run()
     reprec.assertoutcome(passed=1)
+
+
+@pytest.mark.issue(3498)
+@pytest.mark.parametrize("base", [
+    'six.moves.builtins.object',
+    'unittest.TestCase',
+    'unittest2.TestCase',
```

Comment:
We never install `unittest2`, should we add it to our testing dependencies in `tox.ini`?

Reply:
atm i dont think thats necessary